### ISSUE TITLE
WARN is too severe for the "Previous digest authentication with same nonce failed, returning null"

### ISSUE THREAD

**Issue by ylexus** [rburgst/okhttp-digest#88]
These statements are logged at the WARN level. My app operates a zero noise in logs approach and sends alert when WARN or ERROR are logged. Both typically mean something needs an attention of a human. However, these two do not always happen when password is wrong, despite the comment. I can sometimes observe them during normal operation. Also, these are very low level technical messages, and are logged on a critical path, i.e. with a high volume of traffic has a potential to flood the logs. They are not really problems. It feels like these should be logged at the DEBUG level instead. Can this be done please if you agree?

`BasicAuthenticator`:
```java
Platform.get().log( "Previous basic authentication failed, returning null", Platform.WARN,null);
```

`DigestAuthenticator`
```java
// prevent infinite loops when the password is wrong
Platform.get().log("Previous digest authentication with same nonce failed, returning null", Platform.WARN,
                    null);
```

**Comment by ylexus** [comment]
For that matter, the 1nd argument also applies to this:
```java
Platform.get().log("Cached authentication expired. Sending a new request.", Platform.INFO, null);
```

These are per-request messages, in the presence of large number of requests there's a log spam risk.

**Comment by ylexus** [comment]
Seems like OkHttp logging mechanism used only allows INFO or WARN. For your convenience, here's a PR: https://github.com/rburgst/okhttp-digest/pull/89

**Comment by rburgst** [comment]
released 3.1.1

**Comment by rburgst** [comment]
thanks a lot for your contribution
